docs/api-v2-import.json

```diff
@@ -0,0 +1,602 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "version": "2.0.0",
+    "title": "External event import API",
+    "description": "The purpose of this API specification is to facilitate the import of external event data into Scarf."
+  },
+  "servers": [
+    {
+      "url": "http://api.scarf.sh"
+    }
+  ],
+  "tags": [
+    {
+      "name": "External event import",
+      "description": "The external event import API provides a way for you to integrate with other applications and platforms that may be generating event data that  you need to track and manage.\n\nTo start the import process, users can upload the external event data via the dedicated 'importPackageEvents' and 'importTrackingPixelEvents' endpoints\nin the API. These endpoints are specifically designed to receive external event data and kickstart the import process. \n\nTo use the endpoints, users need to provide the events in newline-delimited JSON format (also known as [NDJSON](http://ndjson.org/)). Each event is \nrepresented by a JSON object that can contain any valid key-value pairs. Properties prefixed with '$' are treated specially by Scarf. See the 'Event' properties\nfor more information.\n\nAfter the import process is initiated, it runs asynchronously. The endpoints return an event import ID that can be used to check the status of the import at a later\ntime with the 'getEventImport' endpoint. Additionally, for a detailed per-event report, the  'getImportLogs' endpoint can be consulted. The import log provides information\non any errors or warnings that occurred during the import process.\n\nTo cancel an ongoing import process, the 'abortEventImport' endpoint can be used.\n"
```

Comment:
Small nit: for any variable names like 'getEventImport', let's instead use "`" so we get inline-code highlighting. I tried that locally in these blocks and it works fine.